packages/email-connector/src/index.ts

```diff
@@ -91,9 +91,7 @@ export class EmailConnector extends AbstractConnector {
     }
 
     const provider = this.magic.rpcProvider
-    const account = await provider
-      .enable()
-      .then((accounts: string[]): string => accounts[0])
+    const account = await provider.enable().then((accounts) => accounts[0])
```

Comment:
Note for reviewer: just removing the type fix the build issue as the return payload expected `account?: null | string;`